bin/deku_node.re

```diff
@@ -285,28 +279,54 @@ let node = {
   );
   await({...node, protocol});
 };
-let node = node |> Lwt_main.run;
-let () = Node.Server.start(~initial=node);
+let node = (folder, trusted_validator_membership_change) =>
+  node(folder, trusted_validator_membership_change) |> Lwt_main.run;
+
+let node = folder => {
+  let trusted_validator_membership_change =
+    folder ++ "/trusted-validator-membership-change.json";
+
+  let () =
+    Node.Server.start(
+      ~initial=node(folder, trusted_validator_membership_change),
+    );
+
+  let _server =
+    App.empty
+    |> App.port(Node.Server.get_port() |> Option.get)
+    |> handle_block_level
+    |> handle_received_block_and_signature
+    |> handle_received_signature
+    |> handle_block_by_hash
+    |> handle_protocol_snapshot
+    |> handle_request_nonce
+    |> handle_register_uri
+    |> handle_receive_operation_gossip
+    |> handle_withdraw_proof
+    |> handle_ticket_balance
+    |> handle_trusted_validators_membership(
+         ~persist=Files.Trusted_validators_membership_change.write,
+         ~file=trusted_validator_membership_change,
+       )
+    |> App.start
+    |> Lwt_main.run;
+
+  let (forever, _) = Lwt.wait();
+  let () = Lwt_main.run(forever);
+  ();
+};
+let node = {
+  let folder_node = {
+    let docv = "folder_node";
+    let doc = "The folder where the node lives.";
+    Arg.(required & pos(0, some(string), None) & info([], ~doc, ~docv));
+  };
 
-let _server =
-  App.empty
-  |> App.port(Node.Server.get_port() |> Option.get)
-  |> handle_block_level
-  |> handle_received_block_and_signature
-  |> handle_received_signature
-  |> handle_block_by_hash
-  |> handle_protocol_snapshot
-  |> handle_request_nonce
-  |> handle_register_uri
-  |> handle_receive_operation_gossip
-  |> handle_withdraw_proof
-  |> handle_ticket_balance
-  |> handle_trusted_validators_membership(
-       ~persist=Files.Trusted_validators_membership_change.write,
-       ~file=trusted_validator_membership_change,
-     )
-  |> App.start
-  |> Lwt_main.run;
+  Term.(const(node) $ folder_node);
+};
 
-let (forever, _) = Lwt.wait();
-let () = Lwt_main.run(forever);
+let () = {
+  Random.self_init();
```

Comment:
This needs to be removed if #245 is merged first.

Reply:
please rebase on #245

Reply:
@EduardoRFS rebased